demo.py

```diff
@@ -23,52 +24,91 @@
     'data_format', 'NCHW', 'Data format: NCHW (gpu only) / NHWC')
 tf.app.flags.DEFINE_string(
     'ckpt_file', './saved_model/model.ckpt', 'Checkpoint file')
+tf.app.flags.DEFINE_string(
+    'frozen_model', '', 'Frozen tensorflow protobuf model')
 tf.app.flags.DEFINE_bool(
     'tiny', False, 'Use tiny version of YOLOv3')
 
-tf.app.flags.DEFINE_integer('size', 416, 'Image size')
+tf.app.flags.DEFINE_integer(
+    'size', 416, 'Image size')
 
-tf.app.flags.DEFINE_float('conf_threshold', 0.5, 'Confidence threshold')
-tf.app.flags.DEFINE_float('iou_threshold', 0.4, 'IoU threshold')
+tf.app.flags.DEFINE_float(
+    'conf_threshold', 0.5, 'Confidence threshold')
+tf.app.flags.DEFINE_float(
+    'iou_threshold', 0.4, 'IoU threshold')
+
+tf.app.flags.DEFINE_float(
+    'gpu_memory_fraction', 0.8, 'Gpu memory fraction to use')
```

Comment:
Is this flag required? Maybe rather set it to 1.0 by default

Reply:
I put that flag in just about everything, as I'm often trying to run multiple graphs concurrently. Should definitely default to 1.0. Let me know if you don't think its necessary.